tests/vmi_configuration_test.go

```diff
@@ -2462,10 +2462,12 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 					&expect.BExp{R: "2"},
 				}, 15)).To(Succeed())
 
-				By("Check values in domain XML")
-				domXML, err := libdomain.GetRunningVirtualMachineInstanceDomainXML(virtClient, cpuVmi)
-				Expect(err).ToNot(HaveOccurred(), "Should return XML from VMI")
-				Expect(domXML).To(ContainSubstring("<hint-dedicated state='on'/>"), "should container the hint-dedicated feature")
+				if checks.IsAMD64(testsuite.Arch) {
+					By("Check hint-detected state value in domain XML on amd64")
+					domXML, err := libdomain.GetRunningVirtualMachineInstanceDomainXML(virtClient, cpuVmi)
+					Expect(err).ToNot(HaveOccurred(), "Should return XML from VMI")
+					Expect(domXML).To(ContainSubstring("<hint-dedicated state='on'/>"), "should container the hint-dedicated feature")
```

Comment:
Can we cleanup unit tests (and or production) under `pkg/virt-launcher/virtwrap/api/` (`testdata` as well)

Reply:
Apologies I'm not sure what you're suggesting here?